Data/Generics/Strafunski/StrategyLib/MonadicFunctions.hs

```diff
@@ -25,6 +25,13 @@ newtype Id a = Id a
 unId 		:: Id a -> a
 unId (Id x) 	 = x
 
+instance Functor Id where
```

Comment:
I only added it if the compiler complained. You can try remove it if you like, but pretty sure it is needed